docs/full_conf.py

```diff
@@ -53,7 +53,8 @@
     'macro',
     'macro-def',
     'index',
-    'jira'
+    'jira',
+    'classad-attribute'
```

Comment:
I will note for the record that it would be marginally more Pythonic to have a trailing comma here.